File: php/testcode/benchmark_test_00709.php
<?php
require_once __DIR__ . '/shared.php';

class FileReader030 {
    public string $file = '';
    public function __toString(): string {
        return file_get_contents($this->file);
    }
}

function benchmarkTest00709(BenchmarkRequest $req): BenchmarkResponse {
    $data = $req->post('data');
    $obj = unserialize($data);
    $output = (string) $obj;
    return BenchmarkResponse::ok($output);
}

File: rust/testcode/benchmark_test_00709.rs
pub fn handle(req: &super::shared::BenchmarkRequest) -> super::shared::BenchmarkResponse {
    let input = req.param("text");

    if input.len() > 50 {
        return super::shared::BenchmarkResponse::bad_request("input too long");
    }

    let html = format!("<p>{}</p>", input);

    super::shared::BenchmarkResponse::ok(&html)
}
